jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListPresenter.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.workbench.ht.client.editors.taskslist;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.uberfire.client.annotations.WorkbenchPartTitle;
+import org.uberfire.client.annotations.WorkbenchScreen;
+
+@Dependent
+@WorkbenchScreen(identifier = TaskAdminListPresenter.SCREEN_ID)
+public class TaskAdminListPresenter extends AbstractTaskListPresenter {
+
+    public static final String SCREEN_ID = "Task Admin List";
+
+    @Inject
+    public void setView(final TaskAdminListViewImpl view) {
```

Comment:
Same as the `TaskListPresenter`, you can remove the set view method and add the `TaskAdminListViewImpl` type as parameter to `AbstractTaskListPresenter`